src/nectarchain/data/management.py

```diff
@@ -54,7 +54,14 @@ def findrun(run_number: int, search_on_GRID=True) -> Tuple[Path, List[Path]]:
             if search_on_GRID:
                 log.warning(e, exc_info=True)
                 log.info("will search files on GRID and fetch them")
-                lfns = DataManagement.get_GRID_location(run_number)
+                lfns = DataManagement.get_GRID_location(
+                    run_number, basepath="/vo.cta.in2p3.fr/nectarcam"
+                )
+                lfns.extend(
+                    DataManagement.get_GRID_location(
+                        run_number, basepath="/ctao/nectarcam"
+                    )
+                )
                 DataManagement.getRunFromDIRAC(lfns)
                 list = glob.glob(
                     basepath + "**/*" + str(run_number) + "*.fits.fz", recursive=True
```

Comment:
Maybe we could raise here an exception if files are not found, I just realize that it's missing